pybliometrics/scopus/utils/get_content.py

```diff
@@ -1,10 +1,11 @@
+from typing import Type
 from requests import Session
 from requests.adapters import HTTPAdapter
 from urllib3.util import Retry
 
 from pybliometrics import __version__
 from pybliometrics.scopus import exception
-from pybliometrics.scopus.utils.startup import config
+from pybliometrics.scopus.utils.startup import get_config,get_keys, get_throttling_params, get_config_path
```

Comment:
Is that how `init()` is supposed to work? Each time we make an API request we have to read out the entire config again? I thought the `init()` makes the configuration parameters globally available.

Reply:
I implemented the code like this to allow the user to use different configurations after the import. It enables the user to:
- Use the library without `init()` using the default path
- Change the default configuration to a custom one after `init()`

Reply:
I understand... Let's make it as simple as possible.
The only complexity we need is making the config dir **not** static. Because the `init()` function is called after the `import` we need to somehow be able to change the dir of the configuration post import.